scripts/linux-x86_64/README.md

```diff
@@ -0,0 +1,53 @@
+# Alby Hub on a Linux server
+
+## Requirements
+
+- Linux distribution
+- Runs pretty much on any VPS or server
+
+### Installation (non-Docker)
+
+We have prepared an installation script that installs Alby Hub for you.
+We recommend inspecting the install script and if needed adjusting it or taking inspiration from it for your setup.
+
+As a general good practice we recommend creating a new system user.
+
+    $ adduser albyhub
+
+Run the installation script on your server:
+
+    $ wget https://raw.githubusercontent.com/getAlby/hub/master/scripts/linux-x86_64/install.sh
+    $ ./install.sh
```

Comment:
if we do this, it won't run as the correct user, right? it's not the same as the ssh command below

Reply:
why not?

Reply:
we mix two different users, if the user runs a different command next time they use a different user. In my opinion this is confusing

Reply:
I don't get why they run next time a differnt user?

Reply:
there are 2 commands they can run - if the next time they use a different one based on whether they are ssh'd in or not, it uses a different user. I did this by accident

Reply:
which 2 commands are you talking about and why would the user be different?

Reply:
this is the current version: https://github.com/getAlby/hub/pull/415/files